Fix missing floatingip when calling GetLoadBalancer()

FengyunPan opened the pull request:
If user specify floating-network-id, a floatingip and a vip will
be assigned to LoadBalancer service, So its status contains a
floatingip and a vip, but GetLoadBalancer() only return vip.

**Release note**:
```release-note
GetLoadBalancer() only return floatingip when user specify floating-network-id, or return LB vip.
```

dims commented:
/retest

dims commented:
/test all

FengyunPan commented:
Related to #52609

anguslees commented on pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go:
```diff
@@ -1287,7 +1297,16 @@ func (lb *LbaasV1) GetLoadBalancer(clusterName string, service *v1.Service) (*v1
 	status := &v1.LoadBalancerStatus{}
-	status.Ingress = []v1.LoadBalancerIngress{{IP: vip.Address}}
+
+	if len(vip.PortID) != 0 {
```
Just `if vip.PortID != ""` ?

FengyunPan replied:
Done.

anguslees commented on pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go:
```diff
@@ -536,7 +536,17 @@ func (lbaas *LbaasV2) GetLoadBalancer(clusterName string, service *v1.Service) (
-	status.Ingress = []v1.LoadBalancerIngress{{IP: loadbalancer.VipAddress}}
+
+	portID := loadbalancer.VipPortID
+	if len(portID) != 0 {
```
Just `if portID != ""` ?

FengyunPan replied:
Done.

anguslees commented:
This is a user-visible change, so should have a release-note (imo).

Commit:
Fix missing floatingip when calling GetLoadBalancer()

If user specify floating-network-id, a floatingip be assigned to
LoadBalancer service, So its status contains a floatingip, but
GetLoadBalancer() only return vip.

FengyunPan commented:
/retest

dims commented:
/approve no-issue
/lgtm

FengyunPan commented:
/test pull-kubernetes-bazel-build
/test pull-kubernetes-e2e-kubeadm-gce
